beets/importer.py

```diff
@@ -672,7 +673,7 @@ def manipulate_files(self, move=False, copy=False, write=False,
                     # old paths.
                     item.move(copy, link)
 
-            if write and self.apply:
+            if write and (self.apply or self.choice_flag == action.RETAG):
```

Comment:
After checking this issue again, I have decided to keep the explicit comparison with `RETAG` at the moment, mainly because the behaviour of `RETAG` does not always fall under the `APPLY` choice:

| function                   | follows |
|----------------------------|:-------:|
| chosen_ident()             |   ASIS  |
| imported_items()           |   ASIS  |
| align_album_level_fields() |  APPLY  |
| manipulate_files()         |  ~APPLY |

It seemed to me that merging both on `task.apply()` would end up resulting in more confusion and a harder to follow flow! Again, it's rather subjective, so please let me know if you prefer the other approach.